OloEngine/src/OloEngine/Scene/SceneTransition.cpp

```
@@ -0,0 +1,180 @@
+            }
+            return names;
+        }
+    } // namespace
+
+    bool IsSceneFileExtension(const std::filesystem::path& path)
+    {
+        const std::string ext = LowercaseExtension(path);
+        return ext == ".olo" || ext == ".scene";
+    }
+
+    std::filesystem::path ResolveScenePath(std::string_view request, const std::filesystem::path& rootDirectory)
```

Cppcheck (2.21.0)

[style] The function 'ResolveScenePath' is never used.

(unusedFunction)